crates/rpc/rpc-eth-api/src/helpers/call.rs

```diff
@@ -935,7 +1096,7 @@ pub trait Call: LoadState + SpawnBlocking {
         mut request: TransactionRequest,
         gas_limit: u64,
         db: &mut CacheDB<DB>,
-        overrides: EvmOverrides,
+        overrides: Option<EvmOverrides>,
```

Comment:
hmm, why do we need to make this optional?

Reply:
no real reason for this, `Default::default()` can be passed anyway

Reply:
Updated in fc60e9920